### PR TITLE
sgx-tools: vendor the newest rune package to test both challenger-epid and challenger-ecdsa

### PR DESCRIPTION
**Pull request by hustliyilin**
1. sgx-tools: rename the get-ias-report subcommand as verify-quote subcommand and delete the unused input paramenter.
2. sgx-tools: vendor the newest rune package.
3. sgx-tools: refactor readAndCheckFile as readFile.
4. sgx-tools: both support to verify epid and ecdsa quote.
5. sgx-tools/dist/rpm && deb: add the compile dependence for verifying SGX DCAP quote.
6. CI/CD: update the sgx-tools verify-quote command test cases.
7. sgx-tools/README.md && skeleton/README.md: update the sgx-tools dependence and fix a markdown code style error.

Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

**Comment by hustliyilin** [comment]
This is a test pr for #631 , depend on #631 @jiazhang0 @haosanzi

File: sgx-tools/verify-quote.go [hunk 1]
```diff
@@ -13,24 +13,20 @@ import (
 	"os"
 )
 
-var getIasReportCommand = cli.Command{
-	Name:  "get-ias-report",
-	Usage: "get remote attestation report from IAS",
+var verifyQuoteCommand = cli.Command{
+	Name:  "verify-quote",
+	Usage: "verify quote with the help of IAS",
```
Comment (jiazhang0) [inline sgx-tools/verify-quote.go:17]: Because this is sgx-tools, it can do more things than the others. The generation of IAS report will be still kept in attestation but Enclave TLS will always use quote v2 to support EPID-based attestation and embed quote v2 into TLS cert extension instead of IAS report. The only consumer of IAS report in attestation package will be sgx-tools and shelter. So please keep it.

Comment (jiazhang0) [reply sgx-tools/verify-quote.go:17]: I mean the eventual results would be:
- the verification to quote v2 (EPID) is available
- the verification to quote v3 (ECDSA) is available
- the verification to IAS report is available

**Comment by hustliyilin** [comment]
Need to be refactored.

File: sgx-tools/verify-quote.go (hunk 1 above)
Comment (jiazhang0) [inline sgx-tools/verify-quote.go:17]: "Verify the IAS attestation verification report"

File: sgx-tools/gen-quote.go [hunk 2]
```diff
@@ -72,14 +71,14 @@ For example, generate the quote file according to the given local report file:
...
+			if len(spid) != intelsgx.SpidLength*2 {
+				return fmt.Errorf("Spid must be %d-character long", intelsgx.SpidLength*2)
 			}
 
 			p = parseSgxEpidAttester(quoteType, spid)
-			attestationType = sgx.AttestationEpid
+			attestationType = intelsgx.AttestationEpid
 		} else {
-			attestationType = sgx.AttestationEcdsa
+			attestationType = intelsgx.AttestationEcdsa
```
Comment (jiazhang0) [inline sgx-tools/gen-quote.go:84]: Can you move above changes to another commit?
